collect_app/build.gradle

```diff
@@ -200,6 +200,9 @@ android {
         unitTests {
             includeAndroidResources = true
             returnDefaultValues = true
+            all {
+                maxHeapSize = "1024m"
```

Comment:
Without this my build kept failing it's probably connected somehow to bumping targetSdkVersion https://circleci.com/docs/2.0/language-android/#handling-out-of-memory-errors